core/trino-main/src/main/java/io/trino/dispatcher/DispatchManager.java

```diff
@@ -89,6 +90,7 @@ public DispatchManager(
             AccessControl accessControl,
             SessionSupplier sessionSupplier,
             SessionPropertyDefaults sessionPropertyDefaults,
+            SessionPropertyManager sessionPropertyManager,
```

Comment:
nit: injecting `SessionPropertyManager` can be a separate preparatory commit.

Reply:
done